test/common_nn.py

```diff
@@ -2972,6 +3001,10 @@ def __call__(self, test_case):
         module = self.constructor(*self.constructor_args)
         input = self._get_input()
 
+        if self.expected_output_size is not None:
+            out = test_case._forward(module, input)
+            test_case.assertTupleEqual(out.size(), self.expected_output_size)
```

Comment:
Oh, didn't know this existed... I've been using assertEqual for tuples

Reply:
Yeah the error message for assertEqual is sooo bad. assertTupleEqual is much better